req-travis-conda.txt

```diff
@@ -10,3 +10,5 @@ pyzmq==14.7.0
 scipy==0.16.0
 six==1.9.0
 toolz==0.7.4
+Cython==0.23.4
```

Comment:
Should be fixed in https://github.com/bartvm/mimir/commit/cdac24e2afb912d52c8481a2e6c7342bccddd1e0